pkg/scalers/prometheus_scaler.go

```diff
@@ -134,6 +144,16 @@ func parsePrometheusMetadata(config *ScalerConfig) (meta *prometheusMetadata, er
 		meta.cortexOrgID = val
 	}
 
+	meta.ignoreNullValues = defaultIgnoreNullValues
+	if val, ok := config.TriggerMetadata[ignoreNullValues]; ok && val != "" {
+		if val == "disable" {
```

Comment:
Why don't you just use `true` and `false`? There are other cases with `true/false` and in this case I'd say that the meaning doesn't change

Reply:
Either true/false or enable/disable is ok for me. The enable/disable feels more legit to me when I wrote the code hhh.

Reply:
Let's use true/false to align with other scalers

Reply:
Would that be possible for you?

Reply:
No problem, updated to true/false already